README.md

```diff
@@ -36,25 +43,34 @@ Run a mongo database container in the pod.
 podman run --pod pantheon --name slingmongo -d mongo
 ```
 
+Build the pantheon docker image
+
+```sh
+buildah bud --layers -f container/Dockerfile -t pantheon-app .
+```
```

Comment:
That is in fact the intention here: to run it from the /pantheon directory. The reason is that in order to build the image we need the full project as context (to be able to refer to ../pantheon-slingstart/.... as you mentioned). This is why the `.` at the end of the command line is important: it tells buildah to use the current directory as the context when building the image. But because this directory does not contain a Dockerfile (or Containerfile) we also then need to specify the actual file by using the `-f` parameter.

Reply:
As we discovered during our 1:1, this is likely due to me being on Fedora 28 and having an older version of buildah installed. I'll try upgrading - in the meantime, resolving this.